gst/edge/edge_src.c

```diff
@@ -152,15 +151,10 @@ gst_edgesrc_set_property (GObject * object, guint prop_id, const GValue * value,
 
   switch (prop_id) {
     case PROP_HOST:
-      if (!g_value_get_string (value)) {
-        nns_logw ("host property cannot be NULL");
-        break;
-      }
-      g_free (self->host);
-      self->host = g_value_dup_string (value);
+      nns_logw ("host property is deprecated");
       break;
     case PROP_PORT:
-      self->port = g_value_get_uint (value);
+      nns_logw ("port property is deprecated");
```

Comment:
If TCP, MQTT is used it's not deprecated?

Reply:
Host and port are also unnecessary in TCP and MQTT for edgesrc.
Destination host and port are enough for edgesrc.
For edgesink, host and port are required.